out_mongo.go

```diff
@@ -119,6 +134,48 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 	return output.FLB_OK
 }
 
+func ProcessAll(ctx context.Context, dec *output.FLBDecoder, processor entry.Processor) error {
+	g, ctx := errgroup.WithContext(ctx)
+
+	// For log purpose
+	startTime := time.Now()
+	total := 0
+	logger, err := log.GetLogger(ctx)
+	if err != nil {
+		return fmt.Errorf("get logger: %w", err)
+	}
+
+	// Iterate Records
+	for {
+		total++
+
+		// Extract Record
+		record, err := entry.GetRecord(dec)
+		if err != nil {
+			if errors.Is(err, entry.ErrNoRecord) {
+				logger.Info("Records flushed", map[string]interface{}{
+					"count":    total,
+					"duration": time.Since(startTime),
+				})
```

Comment:
```suggestion
				logger.Debug("Records flushed", map[string]interface{}{
					"count":    total,
					"duration": time.Since(startTime),
				})
```